frontend/src/components/layout/Tabs.vue

```diff
@@ -1,46 +1,78 @@
 <template>
     <div class="w-full rounded-lg bg-white mb-16 shadow overflow-hidden dark:bg-neutral-800">
-        <TabGroup>
-            <TabList class="border-b border-gray-200 px-6 gap-x-4 -mb-px grid dark:border-neutral-700 sm:grid-cols-2 md:grid-
-                            cols-3 lg:grid-cols-none lg:auto-cols-min lg:grid-flow-col">
-                <Tab v-for="(label, key) in labels" as="template" :key="key" v-slot="{ selected }">
-                    <button :class="[
-                        'flex items-center focus:outline-none whitespace-nowrap pt-4 pb-3 px-3 font-medium text-base relative',
-                        'before:block before:z-1 before:w-0 before:absolute before:left-1/2 before:-translate-x-1/2 before:-bottom-[1px] before:h-[3px] before:bg-sky-500 before:hover:w-full before:transition-all',
-                        'after:block after:absolute after:-bottom-[1px] after:h-px after:bg-gray-200 dark:after:bg-neutral-700 after:-left-6 after:-right-6',
-                        { 'border-sky-500 text-sky-600 before:!w-full': selected },
-                    ]">
-                        <component v-if="label.icon" :is="label.icon" class="mr-2"></component>
+        <tab-group>
+            <tab-list
+                class="border-b border-gray-200 px-6 gap-x-4 -mb-px grid dark:border-neutral-700 sm:grid-cols-2 md:grid-
+                            cols-3 lg:grid-cols-none lg:auto-cols-min lg:grid-flow-col"
+            >
+                <tab
+                    v-for="label in props.labels"
+                    :key="label.key"
+                    v-slot="{ selected }"
+                    as="template"
+                >
+                    <button
+                        class="flex items-center focus:outline-none whitespace-nowrap pt-4 pb-3 px-3 font-medium text-base
+                        relative before:block before:z-1 before:w-0 before:absolute before:left-1/2 before:-translate-x-1/2
+                        before:-bottom-[1px] before:h-[3px] before:bg-sky-500 before:hover:w-full before:transition-all
+                        after:block after:absolute after:-bottom-[1px] after:h-px after:bg-gray-200 dark:after:bg-neutral-700
+                        after:-left-6 after:-right-6"
+                        :class="{
+                            'border-sky-500 text-sky-600 before:!w-full': selected
+                        }"
+                        type="button"
+                    >
+                        <component
+                            :is="label.icon"
+                            v-if="label.icon"
+                            class="mr-2"
+                        />
                         {{ label.title }}
-                        <span v-if="label.count !== undefined"
-                            class="ml-2 bg-gray-300 rounded-full px-2.5 py-0.5 text-xs font-medium text-gray-900">
+                        <span
+                            v-if="label.count !== undefined"
```

Comment:
Label counts are gone
<img width="609" alt="Bildschirmfoto 2024-03-09 um 18 48 58" src="https://github.com/FriendsOfShopware/shopmon/assets/412142/e647db60-6ac6-43e7-81cd-b880d7799d85">
<img width="476" alt="Bildschirmfoto 2024-03-09 um 18 49 12" src="https://github.com/FriendsOfShopware/shopmon/assets/412142/31f5a903-fb7f-4410-ab81-94a87f093038">

Reply:
Just forgot this in the column definition